build.gradle

```diff
@@ -20,10 +20,12 @@ repositories {
 
 dependencies {
     compile name: 'touch-image-view'
+    compile 'com.android.support:support-v4:22.2.1'
     compile project(':owncloud-android-library')
     compile 'com.android.support:support-v4:22.2.1'
     compile 'com.jakewharton:disklrucache:2.0.2'
     compile 'com.android.support:appcompat-v7:22.2.1'
+    compile 'com.getbase:floatingactionbutton:1.10.1'
```

Comment:
Speed dial, understood. That's a good reason to keep the dependencies as they are right now.

The point of using FAB from Support Lib was reducing the number of dependency providers, but should not be with the cost of losing functionality.

I wouldn't the classes in 'third_party', keep the dependency in Gradle should be prefered when possible.